Jenkinsfile

```diff
@@ -4,7 +4,7 @@ import ecdcpipeline.PipelineBuilder
 
 project = "python-streaming-data-types"
 
-python = "python3.6"
+python_version = "3.8"
```

Comment:
I don't think this is used, and given that tox tests against several versions is probably just misleading?